lambda_functions/readme.md

```diff
@@ -0,0 +1,19 @@
+# Lambdas and the Layers they use
```

Comment:
do we need any info here on the need to pin urllib3 until we can move to python 3.10?

Reply:
Good point. I'll add a comment about that.